rfcs/README.md

```diff
@@ -2,14 +2,59 @@
 
 We're trying a new approach of checking in Fluent UI RFCs to the repo so that they're more persistent, easier to find, and we can use PR comments to discuss the proposals.
 
+## Why RFCs?
+
+The RFC process is intended to bring a few benefits:
+
+- More efficent and durable decision making
+  - **Efficent** as we can all engage and review on our own time, rather than locking X engineers in a room for hours
+  - **Durable** as we give more time for decisions to be reviewed, commented, and considered. Rather than making a decision
+    after being locked in a room for an hour with X other engineers.
+- A more inclusive review proces
+  - Supporting both quick and deep thinkers
+  - Allowing proposals and comments from contributors
+- Capturing decisions, allowing:
+  - Us to review recent decisions and build ontop of them
+  - Others to join the project more easily
+
+The React RFC Process sums it up nicely[1]
```

Comment:
I think it needs to be like this for the link to work
```suggestion
The [React RFC Process][1] sums it up nicely
```

Reply:
Actually looks like it needs to be:
```
The React RFC Process<sup>[1](#f1)</sup> sums it up nicely
...
- <b id="f1">1</b> - https://github.com/reactjs/rfcs/blob/master/README.md
```

Reply:
Do we really need to render it as a footnote though? Seems like just a normal link would be enough.

Reply:
You're right a link should be preferred, though imo a footnote would be OK as well. I did update it to a link though.